documentation/book/assembly-kafka-mirror-maker-abort-on-send-failure.adoc

```diff
@@ -0,0 +1,17 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-mirror-maker-abort-on-send-failure-{context}']
+
+= Abort Mirror Maker if message sending fails
+
+You can set whether the Mirror Maker container is terminated following an error while sending a message.
+The container is recreated for such a failure on {ProductPlatformName}.
+The error sent is ignored when this option is set to `false`.
```

Comment:
I think it's not clear what "this option" is referring. There is no reference to the option name which is abortonsendfailure.

Reply:
If it's referring to the title, I don't know .. it doesn't sound clear to me.

Reply:
@PaulRMellor can you help us to find better formulation?

Reply:
Good point, Paolo. I suggest rephrasing as follows:

If the `abortonsendfailure` option is set to `false`, message sending errors are ignored.

Reply:
Yeah it sounds better.

Reply:
Fixed.